src/Workspaces/Core/Portable/Workspace/Solution/ProjectInfo.cs

```diff
@@ -367,9 +404,7 @@ internal sealed class ProjectAttributes : IChecksummedObject, IObjectWritable
             /// <summary>
             /// Matches names like: Microsoft.CodeAnalysis.Features (netcoreapp3.1)
             /// </summary>
-            private static readonly Regex s_projectNameAndFlavor = new Regex(@"^(?<name>.*?)\s*\((?<flavor>.*?)\)$", RegexOptions.Compiled);
-
-            private Checksum? _lazyChecksum;
+            private static readonly Regex s_projectNameAndFlavor = new(@"^(?<name>.*?)\s*\((?<flavor>.*?)\)$", RegexOptions.Compiled);
```

Comment:
Holy crap I did not realize we did this. :smile:

Reply:
I think it'd be better if we passed name and flavor as separate parameters